tests/__init__.py

```diff
@@ -0,0 +1,97 @@
+"""
+mdapi
+Copyright (C) 2015-2022 Red Hat, Inc.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+Any Red Hat trademarks that are incorporated in the source
+code or documentation are not subject to the GNU General Public
+License and may only be used or replicated with the express permission
+of Red Hat, Inc.
+"""
+
+import os.path
+
+import requests
+from bs4 import BeautifulSoup
+from requests.exceptions import HTTPError
+
+from mdapi.confdata import servlogr
+from mdapi.database.main import extract_database, fetch_database
+
+"""
+Standard set of databases to run tests against
+"""
+
+LOCATION = "/var/tmp/mdapi-tests/"
```

Comment:
Could you use a randomly generated temp dir instead? Pytest ships with a fixture that makes one for you if you want.

Reply:
I plan on using designated directories for test cache downloads because these caches are comparatively large in size (~1.5GB after extraction); hence, it would be great if they could be reused locally.